docs/source/conf.py

```diff
@@ -117,11 +116,16 @@
 # html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
 # zerovm-sphinx-theme
 
+html_short_title = html_title = "PyDPF-Post"
 html_theme = "pyansys_sphinx_theme"
-html_logo = "_static/pyansys-logo-black-cropped.png"
+html_logo = pyansys_logo_black
```

Comment:
@PProfizi willl you update it for the core too?

Reply:
@cbellot000 Yes, it is in https://github.com/pyansys/pydpf-core/pull/234